cli/src/main/java/com/devonfw/tools/ide/url/model/folder/AbstractUrlFolder.java

```diff
@@ -87,6 +132,18 @@ public Set<String> getChildNames() {
     return this.childNames;
   }
 
+  /**
+   * Will discard all children and reload from the disc.<br>
+   * <br>
+   * <b>ATTENTION:</b> This method is only for a very specific use-case and should only be called when you know exactly what you are doing.
```

Comment:
Please add use case example for this method.

Reply:
removed. Was used before but I then came to another solution. Could be helpful later but I can then readd it whenever needed.